api-server-nest/src/venues/dto/single-venue-response.ts

```diff
@@ -0,0 +1,15 @@
+import {ResponseWrapper} from "../../dto/response-wrapper";
+import isNotNullOrUndefined from "../../utils/is-not-null-or-undefined";
+import {VenueModel} from "../models/venue.model";
+
+export class SingleVenueResponse extends ResponseWrapper {
+    venue: VenueModel
+
+    constructor(copy?: Partial<SingleVenueResponse>) {
+        super(copy)
```

Comment:
I opted to break with the pattern of using that clone utility and just to this a bit more manually, it was causing problems again and is probably over kill for this use case where we are likely never to have more than one or two fields to copy